static/js/admin/src/ResultObjectActionMenu.js

```diff
@@ -192,6 +192,10 @@ define('ResultObjectActionMenu', [ 'jquery', 'jquery-ui', 'StringUtilities',  'A
 			items["editDescription"] = {name : 'Edit Description'};
 		}
 
+		if (metadata.type === 'Work' && $.inArray('editDescription', metadata.permissions) != -1) {
```

Comment:
There's actually a new permission for this:
https://github.com/UNC-Libraries/box-c/blob/main/auth-api/src/main/java/edu/unc/lib/boxc/auth/api/Permission.java#L29
`editAspaceProperties`
I believe it should already be getting returned in the admin UI.